### PR TITLE
Meta: Improve lack of token scopes message

### PR DESCRIPTION
**Pull request by kidonng**
This is used in `quick-repo-deletion`

## Test URLs

[Link to your fork]

## Screenshot

Before:

image

After:

image

### CODE REVIEW

File: source/features/show-whitespace.css [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*
 Can't use this because attributes are strings and calc() won't cast them
-ttps://stackoverflow.com/a/40815884
+https://stackoverflow.com/a/40815884
```
Comment (kidonng) [inline source/features/show-whitespace.css:3]: In response to https://github.com/sindresorhus/refined-github/pull/4833#discussion_r720638114

**Review by fregante — APPROVED** [review]

**Comment by fregante** [comment]
I think perhaps it's best to include a specific "It doesn't include any scopes" message.

**Comment by yakov116** [comment]
@fregante meta?

**Comment by fregante** [comment]
Not quite. The token is related to RG, but the message comes from a feature

**Pull request merged**